pennylane/beta/tapes/tape.py

```diff
@@ -532,7 +534,7 @@ def trainable_params(self):
         automatically excluded from the Jacobian computation.
 
         The number of trainable parameters determines the number of parameters passed to
-        :meth:`~.set_parameters`, :meth:`~.QuantumTape.execute`, and :meth:`~.QuantumTape.jacobian`,
+        :meth:`~.set_parameters`, :meth:`~.execute`, and :meth:`~.QuantumTape.jacobian`,
```

Comment:
The link to `jacobian` will work later on right?

Reply:
yep! By the next PR :)